src/main/resources/db/migration/0350/V393__DeleteShrubTree.sql

```diff
@@ -0,0 +1,14 @@
+-- For all species that currently have the "Shrub/Tree" growth form, insert "Shrub" and "Tree"
+-- growth forms if they aren't already there.
+--
+-- 1 = Tree, 2 = Shrub, 11 = Shrub/Tree
+INSERT INTO species_growth_forms (species_id, growth_form_id)
+SELECT species_id, series
+FROM species_growth_forms,
+     LATERAL generate_series(1, 2) AS series
+WHERE growth_form_id = 11
```

Comment:
Does this mean it's possible that some trees will have both Tree and Shrub in their growth forms but will actually only be one?

Reply:
The "Shrub/Tree" growth form means it can be both, so if the user selected that for a species, having both growth forms will be correct (or at least, it'll be what the user asked for, just in a different format).

Originally we didn't support multiple growth forms per species, so "Shrub/Tree" was a workaround to simulate selecting both.

Reply:
Makes sense. Thanks!